locale/cs/LC_MESSAGES/clipboard-indicator.po

```diff
@@ -13,21 +13,20 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Last-Translator: Daniel Rusek <mail@asciiwolf.com>\n"
+"Last-Translator: Pavel Borecki <pavel.borecki@gmail.com>\n"
 "Language-Team: \n"
-"X-Generator: Poedit 2.0.3\n"
 
 #: extension.js:78
 msgid "Text will be here"
 msgstr "Zde bude text"
 
 #: extension.js:119
 msgid "Type here to search..."
-msgstr "Vyhledávejte psaním sem..."
+msgstr "Vyhledávejte psaním sem…"
 
 #: extension.js:166
 msgid "Private mode"
-msgstr "Soukromý mód"
+msgstr "Stop (proti úniku citlivých údajů)"
```

Comment:
I think the current (literal) translation is much better.

Reply:
I do see from point of view of someone, who already know, what is this function doing and is bored to read that long string. But what about users, who wasn't use this function before? For example, in Firefox there is also similar situation with private mode, but when someone use it, (s)he gets explanation in opened window/tab, so user is not lost and know exactly, what is going on. I tested furthermore with users and result is, that it could be maybe even better to replace word "Stop" with "Pozastavit" (space wise, it fits into menu without problems). Even more explanatory would be: "Pozastavit (proti nechtěnému vložení citlivých údajů)". See, Gnome as whole is intended for wide user base (and mainly at not so experienced users), so be as explanatory as is possible is good thing (see ease of comprehensives of strings for example in Android and other consume platforms).